CHANGELOG.md

```diff
@@ -9,6 +9,8 @@ The format is based on [Keep a Changelog]
 - Remove GOVUK Verify IAS components and from claimant journeys
 - Documentation updates - DFE Signin process
 - Reduce sensitivity of high response time alert
+- Build new Early Career Payments Policy
```

Comment:
@nomasprime I am simply following what has been done before. The new policy creation is distinct and I feel that it should be called out.

I do agree that there is potential duplication, but again are following the processes we have inherited. 
I also think its a good idea, as a CHANGELOG can be given to a client, similar to the sign posting we have with ADR's in the codebase

Reply:
This just doesn't tie up with the actual change which is "Add 'Did you do your NQT the academic year after your ITT?' page to Early Career Payments". It would be a lot easier to reason with, even automate, if the changelog just pulled in a list of commit titles.

"Build new Early Career Payments Policy" is listed as a subset of the change; "Build new Early Career Payments Journey" isn't listed.